README.adoc

```diff
@@ -42,10 +42,10 @@ You can take a look at the link:sonar-project.properties[sonar-project.propertie
 = Documentation
 
 - https://docs.sonarqube.org/latest/analysis/languages/cfamily/[Documentation of the C, C++ and Objective-C plugin and its Build Wrapper]
-- https://sonarcloud.io/documentation/getting-started/azure-devops/[Documentation of using SonarCloud with Azure DevOps]
```

Comment:
I've just noticed that two links point to SQ instead of SC documentation:
l.36 : should be https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-build-wrapper
l. 44: should be https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/
Could you also update them, please?

Reply:
done.